sql/_05_plcsql/_01_testspec/_04_expression/_04_function_call/_03_builtin_function/_05_datetime_function/cases/04_04_03_05_bfn_datetime_timediff.sql

```diff
@@ -14,8 +13,6 @@ begin
     dbms_output.put_line(TIMEDIFF(TO_TIME('12:34:52'), TO_TIME('00:00:00')));
```

Comment:
Type error in the above line. 
TIMEDIFF(TO_DATE('1999-01-11'), TO_TIME('23:59:59'))